recipes/pyscreeze/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.1.13" %}
+{% set name = "PyScreeze" %}
+
+package:
+  name: pyscreeze
+  version: {{ version }}
+
+source:
+  fn: PyScreeze-{{ version }}.tar.gz
+  md5: 5ce611432f38db75b341eec54428530f
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pillow
```

Comment:
it doesn't look like `pillow` is actually required to build the package.

Reply:
ok, done